profiles/enums.py

```diff
@@ -41,3 +41,16 @@ class Labels:
         WORK = _("Work address")
         HOME = _("Home address")
         OTHER = _("Other address")
+
+
+class LoginMethodType(Enum):
+    PASSWORD = "password"
+    OTP = "otp"
+    SUOMI_FI = "suomi_fi"
+    HELSINKIAD = "helsinkiad"
```

Comment:
I think we can remove `HELSINKIAD` since at least by definition Helsinki AD users should not get a Helsinki profile.